File: pregunta 1/recursivo/recursivo.py
def fib(n):
    if n == 0:
        return (0)
    if n == 1:
        return(1)
    return (fib(n-1) + fib(n-2))



print("Ingrese un valor")
n = int(input())
print("La Serie de Fibonacci es: ")
for i in range(n):
    print(fib(i))

File: pregunta 1/recursivo/recursivo.hs
import Control.Monad.Cont

fib :: Integer -> Integer
fib 0 = 0
fib 1 = 1
fib n = fib (n-1) + fib (n-2)

main :: IO ()
main = do
    putStrLn "Introduzca un valor"
    n <- getLine
    putStrLn "La serie de fibonacci es "
    forM_ [0..(read n)-1] $ \i -> do
        let v = fib(i)
        print v